packages/core/src/reducers/lastTypingAt.js

```diff
@@ -0,0 +1,32 @@
+/*eslint no-case-declarations: "off"*/
+/*eslint no-unused-vars: "off"*/
+
+import { INCOMING_ACTIVITY } from '../actions/incomingActivity';
+
+const DEFAULT_STATE = {};
+
+export default function lastTypingAt(state = DEFAULT_STATE, { payload, type }) {
+  switch (type) {
```

Comment:
Using an `if` could simplify the code because we are handling a single case only. But also okay to keep using `switch`. This is very gray area. 🧓🏻